action.yml

```diff
@@ -25,6 +25,10 @@ inputs:
     description: 'The title for the GitHub issue to be created'
     required: false
     default: 'ZAP Scan Baseline Report'
+  fail_action:
+    description: 'Fail or pass the action based on identified alerts'
+    required: false
+    default: false
```

Comment:
Doesnt this change how the action currently works? I think we should stick to the same default behaviour

Reply:
Yes, this changes the behavior, the previously failed alerts will start to pass with the new version. Sure, I'll revert to the previous behavior.

Reply:
Not failing the action by default seems more appropriate, an issue is raised (or updated) with the results so the user is already being informed of the outcome. I'm starting to wonder if we really need to provide this option (and just fail when there's an issue in the script/scan and not in the number of alerts).

Reply:
Agreed, the reason to open this issue was Github continuously sending emails for the failed actions (Even though the alerts are the same).

Reply:
@psiinon any update on this decision ?

Reply:
OK, I'm good with that - we want the scan to be useful and not annoying. Will we still fail the first time?

Reply:
No, but the issue will be raised (so ok?).

Reply:
@thc202 totally agree with you. A fail in this context means a fail with the actual script running, e.g, the action itself fails. In this case it runs so the default should be a pass. 

An improved implementation would be to fail based on a param, being if any high, medium or low issues are found. You could then block merges accordingly. However at the moment its just annoying.

I accept that this is all well and good for me to say as a user and not (yet) a contributor, I'm afraid PRing on this is probably a bit out of my depth but I'm happy to help if I can.